src/debug_utils.cc

```diff
@@ -480,16 +480,15 @@ std::vector<std::string> NativeSymbolDebuggingContext::GetLoadedLibraries() {
            i++) {
         WCHAR module_name[MAX_PATH];
         // Obtain and report the full pathname for each module
-        if (GetModuleFileNameExW(process_handle,
-                                 modules.data[i],
-                                 module_name,
-                                 arraysize(module_name) / sizeof(WCHAR))) {
+        if (GetModuleFileNameW(
+                modules.data[i], module_name, arraysize(module_name))) {
           DWORD size = WideCharToMultiByte(
               CP_UTF8, 0, module_name, -1, nullptr, 0, nullptr, nullptr);
           char* str = new char[size];
           WideCharToMultiByte(
               CP_UTF8, 0, module_name, -1, str, size, nullptr, nullptr);
           list.emplace_back(str);
+          delete str;
```

Comment:
Could even make this one an `std::string` that gets moved into the list, right?

Reply:
I opted to delete the pointer to reduce lines changed, since I'd need std::string's resize() and data() replace the char array. 

I don't mind refactoring to use std::string if preferred. Thoughts?

SN: I just read a helpful article you were featured in! Thanks for the review 😊